tests/regressions/OverlayHeaderHeight-cy.js

```diff
@@ -1,45 +0,0 @@
-describe('Overlay Header Height [00j]', function () {
```

Comment:
Is this test really outdated? Is there no head bar any more? @rcorral @philipnrmn